core/qwk_mail_packet.js

```diff
@@ -279,13 +313,54 @@ class QWKPacketReader extends EventEmitter {
             return cb(Errors.DoesNotExist('No messages file found within QWK packet'));
         }
 
-        const encoding = this.packetInfo.defaultEncoding;
+        const encodingToSpec = 'cp437';
+        let encoding = encodingToSpec;
+
         const path = paths.join(this.packetInfo.tempDir, this.packetInfo.messages.filename);
         fs.open(path, 'r', (err, fd) => {
             if (err) {
                 return cb(err);
             }
 
+            //  Some mappings/etc. used in loops below....
+            //  Sync sets these in HEADERS.DAT: http://wiki.synchro.net/ref:qwk
+            const FTNPropertyMapping = {
+                'X-FTN-AREA'    : Message.FtnPropertyNames.FtnArea,
+                'X-FTN-SEEN-BY' : Message.FtnPropertyNames.FtnSeenBy,
+                'X-FTN-FLAGS'   : Message.FtnPropertyNames
+            };
+
+            const FTNKludgeMapping = {
+                'X-FTN-PATH'    : 'PATH',
+                'X-FTN-MSGID'   : 'MSGID',
+                'X-FTN-REPLY'   : 'REPLY',
+                'X-FTN-PID'     : 'PID',
+                'X-FTN-FLAGS'   : 'FLAGS',
+                'X-FTN-TID'     : 'TID',
+                'X-FTN-CHRS'    : 'CHRS',
+                //  :TODO: X-FTN-KLUDGE - not sure what this is?
```

Comment:
X-FTN-KLUDGE is for otherwise-unrecognized/unsupported FTN control lines.

Reply:
@rswindell Thanks! What is the format?

Reply:
The contents (value) of the header field is everything found between (and not including) the Ctrl-A character and the carriage-return. e.g. X-FTN-KLUDGE: SOMEKLUDGE
Often the kludge will have a colon-terminated field name, not always.

Reply:
@rswindell sorry for all the questions - I guess multiple `X-FTN-KLUDGE` entries are allowed?

Reply:
Yes, multiple would be allowed. It's my way of retaining all the metadata of the original message and being able to reproduce it, even if I don't recognize it.